docs/source/llms/transformers/large-models.rst

```diff
@@ -0,0 +1,238 @@
+      - High
+      - High
+      -
+        .. code-block:: python
+
+          import mlflow
+          import transformers
+
+          pipeline = transformers.pipeline(
+              task="text-generation",
+              model="meta-llama/Meta-Llama-3.1-70B",
+          )
+
+          with mlflow.start_run():
+              mlflow.transformers.log_model(
+                  transformers_model=pipeline,
+                  artifact_path="model",
+              )
+
+    * - :ref:`Memory-Efficient Model Logging <transformers-memory-efficient-logging>`
+      - Log a model by specifying a path to a local checkpoint, avoiding loading the model into memory.
+      - **Low**
+      - High
+      -
+        .. code-block:: python
+
+          import mlflow
+
+          with mlflow.start_run():
+              mlflow.transformers.log_model(
+                  # Pass a path to local checkpoint as a model
+                  transformers_model="/path/to/local/checkpoint",
+                  # Task argument is required for this saving mode.
+                  task="text-generation",
+                  artifact_path="model",
+              )
+
+    * - :ref:`Storage-Efficient Model Logging <transformers-save-pretrained-guide>`
+      - Log a model by saving a reference to the HuggingFace Hub repository instead of the model weights.
+      - High
+      - **Low**
+      -
+        .. code-block:: python
+
+          import mlflow
+          import transformers
+
+          pipeline = transformers.pipeline(
+              task="text-generation",
+              model="meta-llama/Meta-Llama-3.1-70B",
+          )
+
+          with mlflow.start_run():
+              mlflow.transformers.log_model(
+                  transformers_model=pipeline,
+                  artifact_path="model",
+                  # Set save_pretrained to False to save storage space
+                  save_pretrained=False,
+              )
+
+
+.. _transformers-memory-efficient-logging:
+
+Memory-Efficient Model Logging
+------------------------------
+
+Introduced in MLflow 2.17.0, this method allows you to log a model without loading it into memory:
+
+.. code-block:: python
+
+    import mlflow
+
+    with mlflow.start_run():
+        mlflow.transformers.log_model(
+            # Pass a path to local checkpoint as a model to avoid loading the model instance
+            transformers_model="path/to/local/checkpoint",
+            # Task argument is required for this saving mode.
+            task="text-generation",
+            artifact_path="model",
+        )
+
+
+In the above example, we passed a path to local model checkpoint/weight as a model to the :py:func:`mlflow.transformers.log_model()` API, instead of the pipeline instance. MLflow will inspect the model metadata the checkpoint and log the model weight without loading it into memory. This way, you can log a huge model such as LLMs with billions of parameters to MLflow with minimal computational resources.
```

Comment:
```suggestion
In the above example, we pass a path to the local model checkpoint/weight as the model argument in the  :py:func:`mlflow.transformers.log_model()` API, instead of a pipeline instance. MLflow will inspect the model metadata of the checkpoint and log the model weights without loading them into memory. This way, you can log an enormous multi-billion parameter model  to MLflow with minimal computational resources.
```